Extensions/DialogueTree/JsExtension.js

```diff
@@ -496,7 +560,7 @@ module.exports = {
     extension
       .addCondition(
         'Current branch title is',
```

Comment:
See my comment, we should never use these for an internal title. Sadly it's a bit late so this will break the games using it, but anyway we should change it. Better now than later. 
Condition names are important because, even if the user are not seeing them, they are stored in the game files (which can be read and even edited manually, so we should give them names like in a programming language). Choosing a proper name is important, because once it's done it's here forever.

Reply:
(It's also possible that in the future, conditions could be called from expressions, so these condition names *will* be shown to the user)

Reply:
should be "Compare current branch title" ?

Reply:
No that should be "CurrentBranchTitle" :) See the other conditions for example, it's not a user facing string, but the internal name of the condition. Same for the others.

Reply:
ah right, got confused there xD changing it now. Hopefully not many people have started using the extension yet